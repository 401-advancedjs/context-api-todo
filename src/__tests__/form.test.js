import React from 'react';
import Form from '../components/form/form.js';
import renderer from 'react-test-renderer';

describe('<Form />', () => {
  test('basic rendering', () => {
    const mountedHeader = shallow(<Form />);
    expect(mountedHeader.find('form')).toBeTruthy();
  });

  test('rendering follows the snapshot', () => {
    const snapshot = renderer.create(<Form />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});