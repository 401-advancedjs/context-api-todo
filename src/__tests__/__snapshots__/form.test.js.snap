// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> rendering follows the snapshot 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    placeholder="Add To Do List Item"
  />
</form>
`;
